From 70c7831a21dda45690466adbb5583ef4de38aba7 Mon Sep 17 00:00:00 2001
From: Saydi <truong@ionmobility.com>
Date: Mon, 17 Jun 2024 16:23:41 +0700
Subject: disable-x11

Signed-off-by: Saydi <truong@ionmobility.com>
---
 build_overrides/vulkan_headers.gni | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/flutter/build_overrides/vulkan_headers.gni b/flutter/build_overrides/vulkan_headers.gni
index 5efa852..8a4144d 100644
--- a/flutter/build_overrides/vulkan_headers.gni
+++ b/flutter/build_overrides/vulkan_headers.gni
@@ -6,6 +6,6 @@
 # set anything.
 
 if (is_linux) {
-  vulkan_use_x11 = true
+  vulkan_use_x11 = false
   vulkan_use_wayland = true
 }
-- 
2.25.1

