From 649f670ec1237c4e58315981913b1b99e67262b3 Mon Sep 17 00:00:00 2001
From: Saydi <truong@ionmobility.com>
Date: Mon, 17 Jun 2024 16:49:13 +0700
Subject: gn riscv32 and riscv64

Signed-off-by: Saydi <truong@ionmobility.com>
---
 tools/gn | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/flutter/tools/gn b/flutter/tools/gn
index 0be76e5..8a3c8cc 100755
--- a/flutter/tools/gn
+++ b/flutter/tools/gn
@@ -119,7 +119,7 @@ def can_use_prebuilt_dart(args):
       prebuilt = 'macos-x64'
     else:
       prebuilt = 'linux-x64'
-  elif args.target_os == 'linux' and args.linux_cpu in ['x64', 'arm64']:
+  elif args.target_os == 'linux' and args.linux_cpu in ['x64', 'arm64', 'riscv32', 'riscv64']:
     prebuilt = 'linux-%s' % args.linux_cpu
   elif args.target_os == 'mac' and args.mac_cpu in ['x64', 'arm64']:
     prebuilt = 'macos-%s' % args.mac_cpu
@@ -1015,7 +1015,7 @@ def parse_args(args):
   parser.add_argument('--web', action='store_true', default=False)
   parser.add_argument('--windows', dest='target_os', action='store_const', const='win')
 
-  parser.add_argument('--linux-cpu', type=str, choices=['x64', 'x86', 'arm64', 'arm'])
+  parser.add_argument('--linux-cpu', type=str, choices=['x64', 'x86', 'arm64', 'arm', 'riscv32', 'riscv64'])
   parser.add_argument('--fuchsia-cpu', type=str, choices=['x64', 'arm64'], default='x64')
   parser.add_argument('--windows-cpu', type=str, choices=['x64', 'arm64', 'x86'], default='x64')
   parser.add_argument('--simulator-cpu', type=str, choices=['x64', 'arm64'], default='x64')
-- 
2.25.1

